modules/thermal_hydraulics/src/components/FlowChannel1Phase.C

```diff
@@ -26,6 +26,8 @@ FlowChannel1Phase::validParams()
   params.addParam<FunctionName>("initial_T", "Initial temperature in the flow channel [K]");
   params.addParam<FunctionName>("D_h", "Hydraulic diameter [m]");
 
+  params.declareControllable("initial_p initial_T initial_vel D_h");
```

Comment:
I don't think we want to make spatial-dependent quantities controllable. Here, `D_h` is spatial-dependent. Similarly we should not be doing that for 'A' and 'P_hf'. We can change those either via functions or via a transfer.

Reply:
> I don't think we want to make spatial-dependent quantities controllable

Just to clarify, all of these things are currently controllable already. Do you want me to go ahead and make them uncontrollable? 🐯

Reply:
Is that because of the "if-const make controllable" statements we have there?

Reply:
Yep

Reply:
Ok.